packages/nimbus/index.js

```diff
@@ -58,6 +58,9 @@ module.exports = function cli(tool) {
       context.addArg(`./${srcFolder}`);
       context.addOption('--out-dir', context.args.esm ? './esm' : `./${buildFolder}`);
     }
+
+    // Changed in v7.7
+    driver.metadata.filterOptions = true;
```

Comment:
so this filters any options to babel since unknown args like `--clean` or `--esm` now throw? 👍

Reply:
It will filters all options that Babel doesn't support natively.